configs/other/server-environment-startup.json

```diff
@@ -3,7 +3,9 @@
         "serverPort": 3375,
         "serverConfig": "http://localhost:3375/configs/openfin/manifest-local.json",
         "clientRoute": "http://localhost:3375/",
-        "channelAdapter": "openfin"
+        "channelAdapter": "openfin",
+        "e2odebug": true,
```

Comment:
I wonder if it might be worthwhile to simplify the config. Maybe just have one property `"e2odebug"` where if it is true, it doesn't break on start, but if it is set to "break", then it will. 
```
        "channelAdapter": "openfin",
        "e2odebug": true,
```
Or
```
        "channelAdapter": "openfin",
        "e2odebug": "break",
```
Thoughts?